src/material-examples/chips-autocomplete/chips-autocomplete-example.html

```diff
@@ -4,8 +4,8 @@
       *ngFor="let fruit of fruits"
       [selectable]="selectable"
       [removable]="removable"
-      (remove)="remove(fruit)">
-      {{fruit.name}}
+      (removed)="remove(fruit)">
+        {{fruit}}
```

Comment:
I think you've accidentally added extra spaces here.

Reply:
haha. yes it was mine. do i need to remove them ? 😄

Reply:
Yes please

Reply:
Sorry for too late. i'v fixed it ^_^